src/data/make_network_data.py

```diff
@@ -1,19 +1,20 @@
 import argparse
+import gzip
```

Comment:
is there any consensus on ordering? is it alphabetical or, when it's used?

Reply:
Not sure, you can "optimize imports" in pycharm though